src/main/java/com/plesba/datamanager/transformers/NullTransformer.java

```diff
@@ -0,0 +1,72 @@
+package com.plesba.datamanager.transformers;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import java.io.IOException;
+import java.io.PipedInputStream;
+import java.io.PipedOutputStream;
+
+public class NullTransformer {
+    private final PipedInputStream inputStream;
```

Comment:
I don’t understand why one of these streams is declared ‘final’ and the other isn’t. Maybe I am missing something obvious, though.